### PR TITLE
Fix sound_queue

### PR DESCRIPTION
**Pull request by NoSloppy**
`busy_` was always true. This seems to work now.

### CODE REVIEW

File: sound/sound_queue.h [hunk 1]
```diff
@@ -46,6 +46,8 @@ class SoundQueue {
       queue_[0].Play(player.get());
       sounds_--;
       for (int i = 0; i < sounds_; i++) queue_[i] = queue_[i+1];
+      // Reset busy_ when no sounds are left to play
+      if (sounds_ == 0) busy_ = false;
```
Comment (profezzorn) [inline sound/sound_queue.h:50]: We just started playing a sound, how can we be not busy?

Comment (NoSloppy) [reply sound/sound_queue.h:50]: Isn't that once the queue is exhausted and we're done?

Comment (profezzorn) [reply sound/sound_queue.h:50]: busy() should only return true if the sound queue is empty AND no sound is playing.

Comment (profezzorn) [reply sound/sound_queue.h:50]: Fixed elsewhere.

**Pull request closed without merging**